tools/system_libs.py

```diff
@@ -291,7 +291,8 @@ def create_libcxx(libname):
       'utility.cpp',
       'valarray.cpp',
       'variant.cpp',
-      'vector.cpp'
+      'vector.cpp',
+      'experimental/filesystem/path.cpp'
```

Comment:
please use `os.path.join('experimental', 'filesystem', 'path.cpp')` (that way it should work on windows too).

Reply:
It works with the '/' slashes on my Windows machine, but I changed it as you proposed, looks more elegant. Thank you.